meta-mender-toradex-nxp/conf/machine/colibri-imx7-mender.conf

```diff
@@ -56,3 +60,7 @@ MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += "mtd-utils mtd-utils-ubifs"
 # pick u-boot-toradex-fsl-fw-utils as provider of fw_{printenv,setenv}
 PREFERRED_PROVIDER_u-boot-fw-utils ?= "u-boot-toradex-fsl-fw-utils"
 PREFERRED_RPROVIDER_u-boot-fw-utils ?= "u-boot-toradex-fsl-fw-utils"
+
+# Set product id vars (needed by toradex_image_json)
+TORADEX_PRODUCT_IDS_append = " 0033"
+TORADEX_PRODUCT_IDS[0033] = "colibri-imx7"
```

Comment:
Should this also use `?=`?

Reply:
Is it legal to use default assignment (`?=`) when doing  an override append?

Reply:
Ah, I think you're right, it isn't. Never mind then.